### ISSUE TITLE
Need help on running the sentiment detection example

### ISSUE THREAD

**Issue by isteven** [angeloskath/php-nlp-tools#44]
Hi, 

First of all thanks for the library. I'm a beginner in NLP, and would like to use this library for a project.

As of now, I'm trying to run the sentiment detection example, but I got error at 

```php 
$test_set = create_test_set($test);
```

Where do you define this function <code>create_test_set</code>? (FYI I'm running these in Win 7)

Also, I am a bit lost:

The file http://php-nlp-tools.com/files/sentiment-detection/dataset_lists.zip is used for both training & testing the model, right?

And after that, the trained & tested model will be used to determine the sentiment of this file http://www.cs.cornell.edu/people/pabo/movie-review-data/review_polarity.tar.gz, correct?

Thanks!

**Comment by angeloskath** [comment]
Hi,

Thanks for the good words and sorry for the late reply.

The function `create_test_set(...)` should be `create_training_set(...)` and must be fixed. Regarding the [dataset_lists.zip](http://php-nlp-tools.com/files/sentiment-detection/dataset_lists.zip), this file contains a 10-fold split for training and testing which means it contains the file names in the training set and in the test set. The actual data are in the [review_polarity.tar.gz](http://www.cs.cornell.edu/people/pabo/movie-review-data/review_polarity.tar.gz).

Angelos
